src/libmongoc/tests/test-mongoc-sdam-monitoring.c

```diff
@@ -958,6 +958,241 @@ test_no_duplicates (void)
    mongoc_client_pool_destroy (pool);
 }
 
+typedef enum {
+   SERVER_HEARTBEAT_STARTED = 0,
+   SERVER_HEARTBEAT_SUCCEEDED,
+   SERVER_HEARTBEAT_FAILED,
+} smm_event_type_t;
```

Comment:
```suggestion
static const char *SERVER_HEARTBEAT_STARTED = "started";
static const char *SERVER_HEARTBEAT_SUCCEEDED = "succeeded";
static const char *SERVER_HEARTBEAT_FAILED = "failed";
```
Suggest using static strings to ease identifying the values on a failure. Change the assertions below to use `ASSERT_CMPSTR`. This changes the failure message from:


```
Assert Failure: 0 == 1
```

To:

```
Assert Failure:
  "succeeded"
  !=
  "started"
```